src/javascript/ContentManager/ContentRoute/ContentLayout/FilesGrid/FilesGrid.redux-reducer.js

```diff
@@ -1,6 +1,8 @@
+const localStorage = window.localStorage;
+
 const initialState = {
-    mode: 'grid',
-    gridMode: 'thumbnail'
+    mode: localStorage.getItem('cmm_files_selector_mode') !== null ? localStorage.getItem('cmm_files_selector_mode') : 'grid',
```

Comment:
Could you define constants for the keys used with local storage since they are referred to in several file?